src/Engine/Manager/ScriptManager.cpp

```diff
@@ -84,7 +85,8 @@ void AngelScriptDebugLineCallback(asIScriptContext *ctx, const std::map<std::str
 }
 
 void print(const std::string& message) {
-    Log() << message;
+    std::cout << message;
```

Comment:
Revert this change (this should be changed in the logging tool instead. You don't have to include iostream then.